src/Products/Annotation/Annotator/AbstractBoxAnnotator.cs

```diff
@@ -35,8 +35,8 @@ protected override Rectangle GetBox()
             string startPoint = svgPath.Replace("[a-zA-Z]+", "").Split(' ')[0];
             string endPoint = svgPath.Replace("[a-zA-Z]+", "").Split(' ')[1];
             string[] start = startPoint.Split(',');
-            float startX = float.Parse(start.Length > 0 ? start[0].Replace("M", "") : "0", CultureInfo.InvariantCulture);
-            float startY = float.Parse(start.Length > 0 ? start[1].Replace("M", "") : "0", CultureInfo.InvariantCulture);
+            float startX = float.Parse(start.Length > 0 ? start[0].Replace("M", "").Replace(",", ".") : "0", CultureInfo.InvariantCulture);
+            float startY = float.Parse(start.Length > 0 ? start[1].Replace("M", "").Replace(",", ".") : "0", CultureInfo.InvariantCulture);
             string[] end = endPoint.Split(',');
             float endX = float.Parse(end.Length > 0 ? end[0].Replace("L", "") : "0", CultureInfo.InvariantCulture) - startX;
```

Comment:
Add the same logic for the commas replacement with dots and CultureInfo.InvariantCulture as done for the startX and startY